tests/ast/decoder.py

```diff
@@ -445,6 +445,9 @@ def decode_timezone_expr(self, tz_expr: proto.PythonTimeZone) -> Any:
         assert timezone.utcoffset(datetime.now()).total_seconds() == offset_seconds
         return timezone
 
+    def binop(self, lhs, rhs, fn):
```

Comment:
I think my original suggestion was cleaner – no need for the lhs and rhs being passed here, since we always dereference `.lhs` and `.rhs` (if not, then let's just not use this helper for those cases).

Reply:
ok